packages/api/lambdas/process-s3-dead-letter-archive.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const log = require('@cumulus/common/log');
+
+const { s3 } = require('@cumulus/aws-client/services');
+const { getJsonS3Object } = require('@cumulus/aws-client/S3');
+const { getKnexClient } = require('@cumulus/db');
+
+const { writeRecords } = require('./sf-event-sqs-to-db-records');
+
+async function processDeadLetterArchive({
+  knex,
+  bucket = process.env.system_bucket,
+  path = `${process.env.stackName}/dead-letter-archive/sqs/`,
+  writeRecordsFunction = writeRecords,
+  batchSize = 1000,
```

Comment:
Should this be configurable?

Reply:
Mostly there for unit test speed purposes, takes several minutes to spool through 1000-sized S3 pages and run the multiple-batch unit test.